src/fs/rename.js

```diff
@@ -1,5 +1,37 @@
+import {join} from "path";
+import fs, {readFile} from "node:fs/promises";
+import {filesDirPath} from "./filesDirPath.js";
+
 const rename = async () => {
-    // Write your code here 
+  const wrongFilePath = join(filesDirPath,'wrongFilename.txt');
+  const properFilename = join(filesDirPath,'properFilename.md');
+  let properExists = false;
+
+  // check if wrongFile exists
+  try {
+    await readFile(wrongFilePath);
```

Comment:
Please use `access()` instead of `readFile()` when you need to just verify that the file exists. Otherwise, you need to wait for reading when the file size is pretty big.